gremlin-server/src/main/java/org/apache/tinkerpop/gremlin/server/GremlinServer.java

```diff
@@ -108,54 +108,24 @@ public GremlinServer(final Settings settings) {
             workerGroup = new NioEventLoopGroup(settings.threadPoolWorker, threadFactoryWorker);
         }
 
-        serverGremlinExecutor = new ServerGremlinExecutor<>(settings, null, workerGroup, EventLoopGroup.class);
+        serverGremlinExecutor = new ServerGremlinExecutor(settings, null, workerGroup, null);
```

Comment:
Looks like the 4th parameter is no longer needed. Since the PR targets `master/`, it would be fine to remove it altogether, no?